pkgs/development/compilers/idris2/default.nix

```diff
@@ -26,9 +33,9 @@ stdenv.mkDerivation rec {
     ++ lib.optional stdenv.isDarwin "OS=";
 
   # The name of the main executable of pkgs.chez is `scheme`
-  buildFlags = [ "bootstrap-build" "SCHEME=scheme" ];
+  buildFlags = [ "bootstrap" "SCHEME=scheme" ];
 
-  checkTarget = "bootstrap-test";
+  checkTarget = "test";
```

Comment:
Same here, why are we changing from `bootstrap-test` to `test`? (I assume because Idris2 "fixed" the `test` target, but leaving a small comment would help me understand).

Reply:
plus bootstrap-test has been removed